patches/v8/0005-Revert-bazel-Add-hide-symbols-from-release-fast-buil.patch

```diff
@@ -1,45 +0,0 @@
-From d0f3e57286f40413f0a54488a098e92e5d86326a Mon Sep 17 00:00:00 2001
```

Comment:
Should we be renumbering the subsequent patches now that 0005 is gone?

I don't actually know if it matters.